labgrid-web-client/src/app/sidebar/sidebar.component.css

```diff
@@ -0,0 +1,48 @@
+.sidebar-container{
+
+}
+
+#sidebar-header{
+    padding: 20px;
+}
+
+.sidebar-item{
+    display:flex;
+    width:100%;
```

Comment:
Remove the line "width: 100%;" because it makes the sidebar items too long. When removed the scrollbar at the bottom disappears.